utils/forms.js

```diff
@@ -53,6 +54,7 @@ export const submitForm = path => {
                 throw new Error(message)
             }
             setStatus({success: true})
+            trackEvent({ category: 'Form', action: 'Submission', label: path })
```

Comment:
Track a form submission, including the name of the form we're submitting using the `path` argument.